pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 )
 
+// DefaultFieldMaxLength is the standard size for most user-settable string fields in Alerting. Use this for typical string fields, unless you have a special reason not to.
+const DefaultFieldMaxLength = 190
```

Comment:
Dare I ask: Should we create *ANOTHER* migration to support longer alert rule names? This is shorter than a tweet.

Reply:
Maybe, but it's not just rule names either. You can see from how common this constant is, this width is used in a LOT of user-editable fields.

IIRC resizing nvarchar fields is a super slow migration too, since basically every row has to be rewritten. I figured we would hold off on this until someone specifically asked for more length.

We could also move to TEXT in some places rather than resize. I'm not a MySQL expert, but at least in modern Postgres there is little reason to use varchar over TEXT any more. Maybe that's also true for MySQL?